source/unit/Fixture.ooc

```diff
@@ -94,12 +94,21 @@ Fixture: abstract class {
 	expect: static func ~integer (value: Int, constraint: Constraint) {
 		This expect(Cell new(value), constraint)
 	}
+	expect: static func ~long (value: Long, constraint: Constraint) {
+		This expect(Cell new(value), constraint)
+	}
+	expect: static func ~ulong (value: ULong, constraint: Constraint) {
+		This expect(Cell new(value), constraint)
+	}
 	expect: static func ~float (value: Float, constraint: Constraint) {
 		This expect(Cell new(value), constraint)
 	}
 	expect: static func ~double (value: Double, constraint: Constraint) {
 		This expect(Cell new(value), constraint)
 	}
+	expect: static func ~ldouble (value: LDouble, constraint: Constraint) {
+		This expect(Cell new(value), constraint)
+	}
 	expect: static func ~isTrue (value: Bool) {
 		This expect(Cell new(value), is true)
 	}
```

Comment:
Hmm. So we need one overload for each kind of Cover we want to use? Would it be possible to instead something like this?
```ooc
expect: static func (value: T, constraint: Constraint) {
    if (T inheritsFrom?(Object))
        value = Cell new(value)
    if (!constraint verify(value))
        TestFailedException new(value, constraint) throw()
}
```
But maybe generics aren't reliable enough for this sort of thing...